Engineering-Reference/GoogleDestinationAO.md

```diff
@@ -498,52 +482,52 @@ Default Value: 3600000
 
 The timeout (in milliseconds) that will be used for all calls to the Google Workspace services. This should be set to a high value to ensure large transfers can be made, unless there is a specific reason to change it, the default value should be retained.
 
-Change Conditions: ??
+Change Conditions: Timeout value can be changed, not recommended for best performance.
 
 ### Retry Count <a name="tpretryc"></a>
 {: .no_toc }
 Default Value: 20
 
 The number of times a failed transfer will be attempted before aborting.
 
-Change Conditions: ??
+Change Conditions: Retry count value can be changed, not recommended for best performance.
 
 ### Modify Request <a name="tmodreq"></a>
 {: .no_toc }
 Default Value: Off
 
 Remove the 'Expect-100' header from requests (required for use behind some proxies).
 
-Change Conditions: ??
+Change Conditions: When enabled the 'Expect-100' header' will be removed. Only required in specific proxy scenario.
```

Comment:
Maybe add do not change with out support

Reply:
Agreed